dj/api/helpers.py

```diff
@@ -16,7 +16,9 @@
 from dj.errors import DJError, DJException, ErrorCode
 from dj.models import Catalog, Column, Database, Engine
 from dj.models.node import (
+    MissingParent,
```

Comment:
Not related to this PR, but the name difference between `MissingParent` and `NodeMissingParents` stands out.

Reply:
Yeah `NodeMissingParents` is a relationship table between `Node` and its `MissingParent`s. In the discussion in PR [#366](https://github.com/DataJunction/dj/pull/366#discussion_r1118230186) I proposed changing `MissingParent` to `MissingUpstream`. Then the relationship table would be `NodeMissingUpstreams`. Let me know if that sounds better.